doc/DeveloperGuide.md

```diff
@@ -88,6 +88,29 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Renaming Tags
+
+**MSS**
+
+1. User requests to rename existing tag's name
+2. Addressbook shows a list of persons that has the tag to be renamed.
+3. Addressbook prompts the user if they want to carry out the renaming.
```

Comment:
The Use Case seeks to model the interaction between an actor and the system.
Therefore the interaction should be alternating between user and the system, user -> system -> user -> system -> etc
In this case, do you know which part should be modified? =)